nixos/modules/misc/ids.nix

```diff
@@ -275,6 +275,8 @@
       gocd-server = 252;
       terraria = 253;
       mattermost = 254;
+      quagga = 255;
```

Comment:
I'm unfamiliar with this service, what is the reason for assigning it a static id?